tests/integration/iop-remote-integration-test-defaults.yaml

```diff
@@ -4,26 +4,13 @@ kind: IstioOperator
 metadata:
   name: install
 spec:
+  profile: remote
```

Comment:
Nit: in the past we never used `profile:` in integration test yamls, because profiles are a delta from the `default` profile, but the integration test base is different. This works fine since it also disables egress gateway, below, but it's a little confusing since one would not expect the egress gateway to be enabled when `profile: remote` is used. The alternative of repeating the full config is not great either.